### DISCUSSION TITLE
How can I change the floating windows color？

### DISCUSSION THREAD

**Discussion by 0BananaBig0** [neoclide/coc.nvim#3462] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/62785154/140919131-cc91706f-da51-483e-bc75-222f92fbaaac.png)
This color looks not good.
I want to change it. I have try this issue https://github.com/neoclide/coc.nvim/issues/1709 but not work.
Can you tell me how to change it? Thanks a lot!

**Comment by fannheyward** [comment]
Try another colorscheme, I'm using `dracula` and it works great.

**Comment by cd-a** [comment, marked as answer]
See here: https://github.com/neoclide/coc.nvim/discussions/3351
